extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.css

```diff
@@ -19,14 +19,15 @@
   height: 1.5em !important;
   width: 1.5em !important;
   bottom: 0.5em !important;
-  border: 0.125em solid var(--i-amphtml-interactive-option-background-color) !important;
+  box-shadow: 0 0 0 0.125em var(--i-amphtml-interactive-option-background-color) !important;
   background-color: var(--interactive-accent-color) !important;
   border-radius: 50% !important;
   text-align: center !important;
   color: var(--i-amphtml-interactive-option-answer-choice-color) !important;
   font-weight: 700 !important;
   line-height: 1.5em !important;
   z-index: 1 !important;
+  backface-visibility: hidden !important;
```

Comment:
Please add a comment above this. 
Could be something describing the behavior like "Prevents 1 pixel shift on animation complete".